src/App.tsx

```diff
@@ -0,0 +1,34 @@
+import "./App.css";
+import ForecastPage from "./Containers/ForecastPage";
+import HourlyForeCastPage from "./Containers/HourlyForeCastPage";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import { WeatherStateData } from "./Context";
+
+const data = {
+  country: "Japan",
+  city: "Tokyo",
+  time: "1:00PM",
+  day: "Friday",
+  date: "22/04/2021",
+  avatar: "",
+  temperature: "22 F",
+  caption: "clear sky",
+};
+
+function App() {
+  return (
+    <WeatherStateData.Provider value={data}>
+      <Router>
+        <Switch>
+          <Route exact path="/" component={ForecastPage} />
+          <Route
+            path={`${data.city}/${data.day}`}
+            component={HourlyForeCastPage}
```

Comment:
You are creating 7 pages for each day, this is not correct the day should be a query parameter, make the changes as discussed on call